homeassistant/components/suez_water/__init__.py

```diff
@@ -26,3 +32,47 @@ async def async_setup_entry(hass: HomeAssistant, entry: SuezWaterConfigEntry) ->
 async def async_unload_entry(hass: HomeAssistant, entry: SuezWaterConfigEntry) -> bool:
     """Unload a config entry."""
     return await hass.config_entries.async_unload_platforms(entry, PLATFORMS)
+
+
+async def async_migrate_entry(
+    hass: HomeAssistant, config_entry: SuezWaterConfigEntry
+) -> bool:
+    """Migrate old suez water config entry."""
+    _LOGGER.debug(
+        "Migrating configuration from version %s.%s",
+        config_entry.version,
+        config_entry.minor_version,
+    )
+
+    if config_entry.version > SuezWaterConfigFlow.VERSION:
```

Comment:
```suggestion
    if config_entry.version > 2:
```
Better to be specific about referencing which version we talk about